### DISCUSSION TITLE
Add link to plex.tv/link in sidebar

### DISCUSSION THREAD

**Discussion by ghost** [sct/overseerr#2239] [Feature Requests]
I would link to have a link in the sidebar to open the plex.tv device link page. I'm switching my users to their own plex accounts and its one less thing to walk them through once getting overseerr setup.

**Comment by TheCatLady** [comment]
We definitely should **_not_** link to https://plex.tv/link anywhere in Overseerr.

If it's required for device setup, the instructions and URL will be displayed on the screen for the user.  The link on its own is pretty useless anyway, as you need the code from the setup screen.
